recipes/jupyterclearcheck/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is already included in the source distribution so it shouldn't be included here.

Reply:
Deleted. Thanks!